model/event/subscribers.go

```diff
@@ -17,6 +17,15 @@ const (
 	EvergreenWebhookSubscriberType  = "evergreen-webhook"
 	EmailSubscriberType             = "email"
 	SlackSubscriberType             = "slack"
+	NoSubscriberType                = "none"
+)
+
+type UserSubscriptionPreference string
+
+const (
+	PreferenceEmail UserSubscriptionPreference = EmailSubscriberType
+	PreferenceSlack                            = SlackSubscriberType
+	PreferenceNone                             = NoSubscriberType
```

Comment:
This vs empty string. Thoughts? Also, I feel this belongs in model.user

Reply:
Removed + moved